test/internal/laws/Chain.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+var compose = require('./compose');
+var flatMap = require('./flatMap');
+var forall = require('./forall');
+
+
+module.exports = function(equals) {
+  return forall({
+
+    //  m >>= f >>= g = m >>= (f >=> g)
```

Comment:
@scott-christopher suggested a different approach on [Gitter][1]:

> Yep, that's the associativity law.
>
> It looks more similar to other associative laws if you use [Kleisli] arrows (`>=>`) instead of `>>=`
>
> ```haskell
> (f >=> g) >=> h ≡ f >=> (g >=> h)
> ```

I like this! There's no clearer expression of associativity than ``(a `op` b) `op` c = a `op` (b `op` c)``.


[1]: https://gitter.im/sanctuary-js/sanctuary?at=5863a0afaf6b364a29049f16